revolv/project/views.py

```diff
@@ -155,20 +157,64 @@ def get_context_data(self, **kwargs):
         context['GOOGLEMAPS_API_KEY'] = settings.GOOGLEMAPS_API_KEY
         return context
 
+class PostProjectUpdateView(UserDataMixin, UpdateView):
+    model = Project
+    template_name = 'project/post_project_update.html'
+    form_class = forms.PostProjectUpdateForm
 
-class PostFundingUpdateView(UpdateView):
-    """
-    The view to send out post funding updates about a project after it has completed.
+    def dispatch(self, request, *args, **kwargs):
+        self.user = request.user
```

Comment:
You should already have `self.user` and `self.is_authenticated` from `UserDataMixin`